File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String[] result = s.split(" ");
        List<String> newResult = Arrays.stream(result).filter(r -> !r.equals("")).toList();
        return newResult.get(newResult.size() - 1).length();
    }
}

File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string zdanie) {
        int j = 0;
        int last = 1;

        for(int i=0;i<zdanie.length();i++){
        if(zdanie[i]!=' '){
            j++;
        }
        else{
            if (j!=0){
                last=j;
            }
            j=0;
        }
    }
    if(j==0){
        return last;
    }
   
    return j;
         

      
    }
};